Issue with bulding

Hello, I am on arch linux and when I try to run the build.sh, i get this error. I have an rtx 3070 and both cuda and cudnn installed

`The Meson build system
Version: 1.1.0
Source dir: /home/shahuda/lc0
Build dir: /home/shahuda/lc0/build/release
Build type: native build
Project name: lc0
Project version: undefined
C++ compiler for the host machine: c++ (gcc 13.1.1 "c++ (GCC) 13.1.1 20230429")
C++ linker for the host machine: c++ ld.bfd 2.40.0
Host machine cpu family: x86_64
Host machine cpu: x86_64
Has header "optional" : YES 
Has header "string_view" : YES 
Has header "charconv" : YES 
Compiler for C++ supports arguments -march=native: YES 
Program scripts/compile_proto.py found: YES (/home/shahuda/lc0/scripts/compile_proto.py)
Program git found: YES (/usr/bin/git)
WARNING: You should add the boolean check kwarg to the run_command call.
         It currently defaults to false,
         but it will default to true in future releases of meson.
         See also: https://github.com/mesonbuild/meson/issues/9300
Message: updating git submodule libs/lczero-common
Message: Using build identifier "git.eed9f57".
Configuring build_id.h using configuration
Run-time dependency threads found: YES
Library dl found: YES
Found pkg-config: /usr/bin/pkg-config (1.8.1)
Found CMake: /usr/bin/cmake (3.26.3)
Run-time dependency tensorflow_cc found: NO (tried pkgconfig and cmake)
Run-time dependency accelerate found: NO (tried pkgconfig and cmake)
Library mkl_rt found: NO
Library mklml found: NO
Library dnnl found: NO
Library openblas.dll found: NO
Library openblas found: NO
Library OpenCL found: YES
Run-time dependency opencl found: NO 
Has header "CL/opencl.h" : YES 
Library cublas found: YES
Library cudnn found: YES
Library cudart found: YES
Program nvcc found: YES (/opt/cuda/bin/nvcc)
Run-time dependency appleframeworks found: NO (tried framework)
Run-time dependency zlib found: YES 1.2.13
WARNING: find_library('libatomic') starting in "lib" only works by accident and is not portable
Library libatomic found: YES
Run-time dependency GTest found: YES 1.13.0
Build targets in project: 10

lc0 undefined

  User defined options
    buildtype: release
    prefix   : /usr/local
    blas     : false

Found ninja-1.11.1.git.kitware.jobserver-1 at /usr/bin/ninja
WARNING: Running the setup command as `meson [options]` instead of `meson setup [options]` is ambiguous and deprecated.
INFO: autodetecting backend as ninja
INFO: calculating backend command to run: /usr/bin/ninja -C /home/shahuda/lc0/build/release
ninja: Entering directory `/home/shahuda/lc0/build/release'
[46/160] Compiling C++ object lc0.p/src_utils_configfile.cc.o
FAILED: lc0.p/src_utils_configfile.cc.o 
c++ -Ilc0.p -I. -I../.. -I../../src -I/usr/include -I/opt/cuda/include -I../../src/neural/cuda -flto=auto -fdiagnostics-color=always -DNDEBUG -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=c++17 -O3 -march=native -DUSE_CUDNN -DNO_PEXT -pthread -isystem../../third_party -MD -MQ lc0.p/src_utils_configfile.cc.o -MF lc0.p/src_utils_configfile.cc.o.d -o lc0.p/src_utils_configfile.cc.o -c ../../src/utils/configfile.cc
In file included from ../../src/utils/configfile.cc:34:
../../src/utils/filesystem.h:45:1: error: ‘uint64_t’ does not name a type
   45 | uint64_t GetFileSize(const std::string& filename);
      | ^~~~~~~~
../../src/utils/filesystem.h:1:1: note: ‘uint64_t’ is defined in header ‘<cstdint>’; did you forget to ‘#include <cstdint>’?
  +++ |+#include <cstdint>
    1 | /*
[59/160] Generating cuda fp16 code with a custom command
ninja: build stopped: subcommand failed.`

--- a/src/utils/filesystem.h
+++ b/src/utils/filesystem.h
@@ -27,7 +27,8 @@
 
 #pragma once
 
-#include <time.h>
+#include <cstdint>
+#include <ctime>
 #include <string>
 #include <vector>
 

--- a/src/version.h
+++ b/src/version.h
@@ -29,6 +29,7 @@
 // Versioning is performed according to the standard at <https://semver.org/>
 // Creating a new version should be performed using scripts/bumpversion.py.
 
+#include <cstdint>
 #include <string>
 #include "version.inc"
 #include "build_id.h"

